gandi/data_gluerecord.go

```diff
@@ -0,0 +1,46 @@
+package gandi
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceGlueRecord() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"zone": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Domain name",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Host name of the glue record",
+			},
+			"ips": {
```

Comment:
Unsure if the maintainers would like to use consistent language across the dataSource ("ips" here) and resource ("values" [below](https://github.com/go-gandi/terraform-provider-gandi/pull/73/files#diff-ffa57e2a4af1be6b765bc7cbc1e1a9759533c5e6c9dc9a3330223ea4327131ffR36))